src/cli/java/org/commcare/util/screen/EntityListSubscreen.java

```diff
@@ -215,9 +215,6 @@ public boolean handleInputAndUpdateHost(String input, EntityScreen host, boolean
                 host.setPendingAction(actions.elementAt(chosenActionIndex));
                 return true;
             }
-        } else if (host.getAutoLaunchAction() != null && allowAutoLaunch) {
-            host.setPendingAction(host.getAutoLaunchAction());
-            return true;
```

Comment:
Can you say more about why this isn't needed anymore.

Reply:
We were earlier relying on this to set the pending auto launch action to the Entity Screen through the `menuSession.handleInput` pathway but since we are directly setting it now [in here](https://github.com/dimagi/formplayer/pull/1251/files) instead of calling `handleInput` we don't need it anymore.